IfcDiscreteAccessory Rail Import BIMQ 2021-07-21T07:41:33.5120880Z (#187)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DiscreteAccessoryTypeSlidingChair/Documentation.md

> Properties of sliding chair t used in railway. The property set can be used by the predefined type SLIDINGCHAIR of IfcDiscreteAccessory

Reviewer: What is "chair t"? Typo?
```suggestion
Properties of sliding chair t used in railway. The property set can be used by the predefined type SLIDINGCHAIR of IfcDiscreteAccessory.
```

> Properties of sliding chair t used in railway. The property set can be used by the predefined type SLIDINGCHAIR of IfcDiscreteAccessory.

Reviewer 2: ```suggestion
Properties of a sliding chair. The property set can be used by the predefined type SLIDINGCHAIR of IfcDiscreteAccessory.
```
dont think we need the `used in railway` bit
